archetypes/helidon/src/main/archetype/mp/oci/files/spec/README.md

```diff
@@ -0,0 +1,5 @@
+# OpenAPI Specification Document
+
+This module contains an [OpenApi v3.0.3 specification](https://spec.openapis.org/oas/v3.0.3) document in the form of
```

Comment:
We should probably remove "v3.0.3" from the visual description since that is bound to go stale quickly.

Reply:
Updated